x-pack/plugin/ccr/qa/src/main/java/org/elasticsearch/xpack/ccr/ESCCRRestTestCase.java

```diff
@@ -217,9 +217,23 @@ protected static void verifyCcrMonitoring(final String expectedLeaderIndex, fina
     }
 
     protected static void verifyAutoFollowMonitoring() throws IOException {
-        Request request = new Request("GET", "/.monitoring-*/_search");
+        Request request = new Request("GET", "/.monitoring-*/_count");
```

Comment:
I wonder if it is worth replacing this with aggregation query?

Reply:
I think that counting is more straightfoward: previous code was searching and summing the first 10 top docs to only compare if the sum was greater than or equal to 1. I think counting achieve the same goal but is much lighter.